examples/README.md

```diff
@@ -0,0 +1,160 @@
+# Examples
+This directory contains examples for `react-native-video` - this is a guide that will show you what can you find here and how to use them.
+
+## Examples Structure
+
+- **[`bare`](#bare)** - Main example ([react-native-test-app](https://github.com/microsoft/react-native-test-app) - bare react-native app) that you can run on: iOS, Android, Windows, visionOS
+
+- **[`expo`](#expo)** - Expo example that you can run on: iOS, Android, tvOS, web (support coming soon)
+
+### Updating Examples Content
+
+Both of applications have mostly the same code (Windows and tvOS have platform-specific code). Other platform are using codebase from `bare` example. 
+If you want to update examples content, you should do it in `bare` example. `expo` example is copping (and overwriting!) `src` folder from `bare` on dependency install. 
+If you want to sync `expo` example, you can use `yarn update-src` command in `expo` example directory.
+
+## How To Run Examples
+
+## [Bare](https://github.com/TheWidlarzGroup/react-native-video/tree/master/examples/bare)
+
+### Configuration
+
+You can configure the example by changing the settings of expo-plugin `app.json` file in the `bare` directory.
```

Comment:
> expo-plugins in bare directory? 
[Test App support expo plugins](https://github.com/microsoft/react-native-test-app/wiki/Config-Plugins) this will make our life easier

Reply:
Got it. Could we mention this in the readme? It seems strange to talk about expo-plugins for an app that doesn't use Expo